jobserv/notify.py

```diff
@@ -132,6 +134,35 @@ def notify_build_complete(build, to_list):
     _send(msg)
 
 
+def notify_build_complete_webhook(build, webhook_url, secret):
+    # try to validate URL certificate before sending POST?
+    # add secret header
+    headers = {
+        'X-JSrv-Signature-256 ': hashlib.sha256(secret).hexdigest()
```

Comment:
I think github has a slightly better approach for this:

https://github.com/foundriesio/jobserv/blob/master/jobserv/api/github.py#L137-L147